apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/peer/endpoint/GetShardingInfo.java

```diff
@@ -56,7 +56,8 @@ public JSONStreamAware processRequest(JSONObject request, Peer peer) {
         }
         log.debug("allShardInfo = [{}], rq.full? = {}", res.shardingInfo.shards.size(), rq.full) ;
         if( rq.full ){ //add list of known peers
-            for(Peer p: Peers.getAllPeers()){
+//            for(Peer p: Peers.getAllPeers()){ too many peers that we can't connect
```

Comment:
Clean that up, pls.

Reply:
No, let is stay to not forget to do something else with it